app/client/components/identity/MarketingPreference.tsx

```diff
@@ -101,6 +113,7 @@ export const MarketingPreference: FC<MarketingPreferenceProps> = props => {
           position: "relative"
         }
       ]}
+      data-link-name={identityName ? linkName : null}
```

Comment:
should this be...
```suggestion
      data-link-name={identityName ? linkName : undefined}
```
... I have no idea, just stood out to me? (i think if `undefined` it's the equiv. of not specifying the attribute)